hardware/Netatmo.cpp

```diff
@@ -443,6 +443,37 @@ bool CNetatmo::ParseDashboard(const Json::Value &root, const int DevIdx, const i
 
 	int batValue = GetBatteryLevel(ModuleType, battery_percent);
 
+	std::time_t tNetatmoLastUpdate = 0;
+	std::time_t tNow = time(NULL);
+
+	// initialize the relevant device flag
+	if ( m_bNetatmoRefreshed.count(ID) == 0 )
```

Comment:
Should this not be
`if ( m_bNetatmoRefreshed.find(ID) == m_bNetatmoRefreshed.end())`

Reply:
I understand from litterature (I am not a c guy) that both methods to test existence of a key are widely accepted as equivalent, but if you perfer .find == .end(), this is fine for me.

Reply:
Yes i would like to have the find option please... i think it's the clearest and fastest way...
Found some small discussion about it as well

https://stackoverflow.com/questions/25490357/checking-for-existence-in-stdmap-count-vs-find

i normally go even further and use/update the resulting item of the find command instead of writing into the map like this

`m_bNetatmoRefreshed(ID)=true`

In this case it's a very simple Boolean map, but if it contains a structure, it's easier to find and update the element